config/enabled.yml

```diff
@@ -714,6 +714,12 @@ Style/UnneededPercentQ:
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#percent-q'
   Enabled: true
 
+Style/UnneededUnderscore:
```

Comment:
The name of the cop is pretty confusing. I'm not sure what would be a better name those - maybe something like `RedundantVariable` or something. @jonas054 ideas welcome!